search-mapper/src/main/java/org/infinispan/search/mapper/log/impl/Log.java

```diff
@@ -19,17 +17,9 @@
 
    int ID_OFFSET_1 = 750000;
 
```

Comment:
It think we can safely re-arrange the IDs without having gaps, because this project has not seen a final release yet. Also the start of the range (750000) might be unsuitable, as per the already existing TODO

Reply:
I'll move the logging ids here to 14501-14800, a range that will be taken away from query module.
My question is now why do we have a separate ```infinispan-search-mapper``` module @gustavonalle , @fax4ever . In my opionion this code can be placed in the ```query``` module.

Reply:
Hi @anistor. Having a separate `infinispan-search-mapper` module that didn't depend on Infinispan helped us with the integration. I think that it can become a package of the query module at any moment...

Reply:
cc @yrodiere

Reply:
Yes the main point of having a separate module was to make initial work easier, since we wouldn't have to care about existing code.

I suppose we could move everything to the same module, now. It's just a matter of what feels more "tidy" to you.

Reply:
Less modules is tidier, especially since infinispan-search-mapper is basically an implementation detail of infinispan-query module, including the HSearch annotations. I'll move search mapper to query in a new PR today if you agree. Or maybe you have pending work on your branches that will be hindered by this move and the move should be postponed @fax4ever , @yrodiere ?

Reply:
No large pending work on our side, no. Feel free to go ahead :)